aleph/views/dashboard_api.py

```diff
@@ -0,0 +1,14 @@
+from flask import Blueprint, request, abort
+
+from aleph.logic.dashboard import get_active_collection_status
+from aleph.views.util import jsonify
+
+blueprint = Blueprint('dashboard_api', __name__)
+
+
+@blueprint.route('/api/2/dashboard', methods=['GET'])
+def dashboard():
+    if request.authz.is_admin:
```

Comment:
Alternatively, we could make this read `request.authz.collections` and intersect the collections the user is able to see with the status board. We're gonna need to serve out collection metadata (like their label) anyway to render this as a GUI.